src/e3/anod/spec.py

```diff
@@ -322,10 +322,11 @@ def build_space_name(self) -> str:
             build_space name otherwise.
         :rtype: str | None
         """
-        if self.enable_name_generator:
-            return self.qualifiers_manager.build_space_name
-        else:
-            return self.name
+        return (
```

Comment:
just use a if block to make code more readable

Reply:
Done